frontend/server/minio-helper.ts

```diff
@@ -37,6 +38,25 @@ export interface MinioClientOptionsWithOptionalSecrets extends Partial<MinioClie
  * @param config minio client options where `accessKey` and `secretKey` are optional.
  */
 export async function createMinioClient(config: MinioClientOptionsWithOptionalSecrets) {
+  // This logic is AWS S3 specific
+  if (isS3Endpoint(config.endPoint)) {
```

Comment:
is endpoint always set when running on an AWS by default (I dont remember us setting this as of now)?
or this becomes a required field now (this would be a breaking change)?

Reply:
When `createMinioClient` is called in the `s3` case it passes in the aws `artifactsConfig` which contains endpoint, that resolves to the default value of `s3.amazonaws.com` or the env variable of `AWS_S3_ENDPOINT.` I'm not sure what you mean by breaking change?

Reply:
https://github.com/kubeflow/pipelines/blob/88a9a31d86d34e61ac0cce9ed22f3455d91c5427/frontend/server/configs.ts#L128

Got it, thanks